### PR TITLE
add support for more Measurements

### PR DESCRIPTION
**Pull request by kb1900**
Fixes https://github.com/coddingtonbear/python-myfitnesspal/issues/110

DEFAULT_MEASURE_AND_UNIT
KeyError: 'fiber', 
KeyError: ''potass.' 

Please let me know if I am doing anything incorrectly - this is my first PR. thanks in advance @coddingtonbear . i hope to also make some contributions to your other repos soon. 

cheers

### CODE REVIEW

File: myfitnesspal/client.py [hunk 1]
```diff
@@ -44,6 +44,9 @@ class Client(MFPBase):
         "protein": (Mass, "g"),
         "sodium": (Mass, "mg"),
         "sugar": (Mass, "g"),
+        "fiber": (Mass, "g"),
+        "potass.": (Mass, "mg"),
+        "kilojoules": (Energy, "Calorie"),
```
Comment (coddingtonbear) [inline myfitnesspal/client.py:49]: I _believe_ this is the wrong mapping here -- that'll equate a kJ with a kcal; I think you might get the correct results here if you  change `Calorie` to `kJ`, but I haven't tested that!

File: myfitnesspal/client.py [hunk 2]
```diff
@@ -145,7 +148,13 @@ def _get_user_metadata(self) -> types.UserMetadata:
             "step_sources",
         ]
         query_string = parse.urlencode(
-            [("fields[]", name,) for name in requested_fields]
+            [
```
Comment (coddingtonbear) [inline myfitnesspal/client.py:151]: I understand that editors commonly autoformat these days, but could I convince you into removing these changes from this PR?  It appears that your reformatting changes disagree with the standard `black` format.

**Review by coddingtonbear — CHANGES_REQUESTED** [review]

**Comment by coddingtonbear** [comment]
Thanks for taking the time to post a PR!  There're just a couple little changes above that I wonder if I could convince you into making before we merge this.  Let me know if you need a hand with any of that, though!

Cheers!

File: myfitnesspal/client.py (hunk 1 above)
Comment (kb1900) [reply myfitnesspal/client.py:49]: Hey you're totally right - I just checked and it is indeed in kJ. I honestly thought they had some mistake on their end but it was me! Will fix this asap.

File: myfitnesspal/client.py (hunk 2 above)
Comment (kb1900) [reply myfitnesspal/client.py:151]: That is weird - I always use black but this is the first time I'm using the black extension with vscode. I wonder if I have some conflicting autoformatting. Thank you for letting me know

**Comment by kb1900** [comment]
Hey @coddingtonbear I believe I made both changes correctly if you could please re-review when you get the chance. Thanks once again for the feedback here!


Only uncertainty I have is that I was already using black for formatting and running the command on my code from command line or within sublime or vs code all resulted in the same changes. I must have some config issue somewhere I will figure out for the future!

**Review by coddingtonbear — APPROVED** [review]
Awesome; thanks for the fix & sorry for the delay!

**Pull request merged**